hash_practice/exercises.py

```diff
@@ -1,19 +1,70 @@
 
+def to_dict(word):
+    anagram = {}
+
+    for letter in word:
+        if letter not in anagram:
+            anagram[letter] = 1
+        else:
+            anagram[letter] += 1
+
+    return anagram
+
+
 def grouped_anagrams(strings):

```

Comment:
✨ Nice! If you want to include the time complexity of sorting/joining the word we can say the time complexity is O(n m log(m)) where n is the length of `strings` and m is the length of `word`. We could also just say time complexity is O(n) and assume strings are all valid English words which would limit their length.